README.md

```diff
@@ -0,0 +1,216 @@
+# xylem
+
+Library to interact with CarbonCulture's datastore
+
+
+## Installation
+
+Installation using pip::
+
+    pip install https://github.com/CarbonCulture/xylem/archive/v0.4.11.zip
```

Comment:
I'm not sure why GitHub doesn't give us a "latest" link or button for xylem on https://github.com/CarbonCulture/xylem/releases, as it does for other repos.. might be nice to have this if its a small fix
?